modules/configuration/ajax/updateProject.php

```diff
@@ -67,8 +67,11 @@
 // Compare submitted values with DB values.
 // It's important not to delete and reinsert the values due to delete cascades on
 // tables referencing project_subproject_rel in the database.
-$toAdd    = array_diff($_POST['SubprojectIDs'], $preValues);
-$toRemove = array_diff($preValues, $_POST['SubprojectIDs']);
+
+if (isset($_POST['SubprojectIDs'])) {
+    $toAdd    = array_diff($_POST['SubprojectIDs'], $preValues);
+    $toRemove = array_diff($preValues, $_POST['SubprojectIDs']);
+}
 
 foreach ($toAdd as $sid) {
```

Comment:
Should line 76 to 94 be moved inside the `if (isset($_POST['SubprojectIDs'])) {..}` or otherwise the $toAdd & $toRemove could possibly be null if isset returns false.